documentation/source/install/cloud_solutions.rst

```diff
@@ -257,76 +257,94 @@ Minimum requirements for container cluster are as follows:
+
+a. Initialize credentials to allow kubectl to execute commands on the container cluster.
 ::
 
     gcloud container clusters list
     gcloud container clusters get-credentials demo-openebs03 --zone us-central1-a
 
-2. Setup the administration context.
-
-* Access the credentails from **Google Cloud Platform** -> **Container Engine** -> **(cluster)** -> **Show Credentials**.
-* Save the *Cluster CA Certificate* to *~/.kube/admin.key*.
-* Create a administration configuration context from the configuration shell using the following commands.
+b. Setup the administrator context.
 
+Create an administrator configuration context from the configuration shell using the following commands.
 ::
 
     gcloud container clusters list
-    kubectl config set-context demo-openebs03 --cluster=gke_strong-eon-153112_us-central1-a_demo-openebs03 --user=cluster-a
+    kubectl config set-context demo-openebs03 --cluster=gke_strong-eon-153112_us-central1-a_demo-openebs03 --user=cluster-admin
 
 The following commands will prompt you for a username and password. Provide username as *admin*. Password for the admin can be obtained from **Google Cloud Platform** -> **Container Engine** -> **(cluster)** -> **Show Credentials**
 ::
 
     kubectl config use-context demo-openebs03
-    kubectl apply -f openebs-operator.yaml
-    kubectl config use-context gke_strong-eon-153112_us-central1-a_demo-openebs03
+    kubectl config use-context gke_strong-eon-153112_us-central1-a_demo-openebs03    
 
-Add OpenEBS related storage classes, that can then be used by developers and applications using the following command.
+Download the latest OpenEBS Operator files using the following commands.
 ::
 
+    git clone https://github.com/openebs/openebs.git
+    cd openebs/k8s
+
+Apply OpenEBS Operator and add related OpenEBS Storage Classes, that can then be used by developers and applications using the following command.
+::
+
+    kubectl apply -f openebs-operator.yaml
     kubectl apply -f openebs-storageclasses.yaml
 
 **Note:**
 
-The persistent storage is carved out from the space available on the nodes (default host directory : */var/openebs*). Development is in progress to provide administrator with additional options of consuming the storage (as outlined in *openebs-config.yaml*). These are slated to work hand-in-hand with the local storage manager of Kubernetes that is due in Kubernetes 1.7/1.8.
+Persistent storage is created from the space available on the nodes (default host directory : */var/openebs*). Development is in progress to provide administrator with additional options of consuming the storage (as outlined in *openebs-config.yaml*). These are scheduled to work hand-in-hand with the local storage manager of Kubernetes that is due in Kubernetes 1.7/1.8.
```

Comment:
due in Kubernetes 1.7/1.8.

Kubernetes 1.7 and 1.8 are already released. This sentence needs to change.

Reply:
Done!